jmxtrans-output/jmxtrans-output-core/src/test/java/com/googlecode/jmxtrans/model/output/OpenTSDBWriterTests.java

```diff
@@ -106,7 +95,7 @@ private static OpenTSDBWriter getOpenTSDBWriter(OutputStream out, List<String> t
 
 	@Test
 	public void writeSingleResult() throws Exception {
-		Server server = Server.builder().build();
+		Server server = Server.builder().setPid("1").build();
```

Comment:
Precondition in Server are requiring to have a pid or a url. I picked the pid. Removing this will mean removing the preconditions